### PR TITLE
New Template: Prometheus Exporter

### PR DESCRIPTION
**Pull request by MarcDouguedroit**

### CODE REVIEW

**Comment by MarcDouguedroit** [comment]
> @MarcDouguedroit , what about adding a link to the documentation. Something like this:
> 
> For more information about the Prometheus Exporter Template, refer to [Consuming Prometheus Metrics in Monitoring Studio X](https://www.sentrysoftware.com/library/swsyx/consuming-prometheus-metrics-in-patrol.html)

@iguitton Yes, but only if we can provide a permanent link to the documentation, one that we know will never change ;-)

**Comment by Raphyyyy** [comment]
The template is working fine, I don't know if @iguitton want to add a link to our documentation (In my opinion: this is a good idea)

**Comment by iguitton** [comment]
> 
> 
> > @MarcDouguedroit , what about adding a link to the documentation. Something like this:
> > For more information about the Prometheus Exporter Template, refer to [Consuming Prometheus Metrics in Monitoring Studio X](https://www.sentrysoftware.com/library/swsyx/consuming-prometheus-metrics-in-patrol.html)
> 
> @iguitton Yes, but only if we can provide a permanent link to the documentation, one that we know will never change ;-)

@MarcDouguedroit / @Raphyyyy , the latest version of the Monitoring Studio X documentation is always stored in https://www.sentrysoftware.com/library/swsyx/. If I make sure we do not rename the markdown file, the URL I provided will always be up-to-date

**Review by bertysentry — DISMISSED** [review]

**Comment by bertysentry** [comment]
@Raphyyyy @NassimBtk @RazeemM Please test and approve :-)

File: templates/prometheus-exporter/prometheusExporterTemplate.cfg [hunk 1]
```diff
@@ -3,7 +3,7 @@ PATROL_CONFIG
 "/SENTRY/X/prometheusExporter/collectionErrorCountAutoAcknowledgeTime" = { REPLACE = "900" },
 "/SENTRY/X/prometheusExporter/credentialsList" = { REPLACE = "" },
 "/SENTRY/X/prometheusExporter/databaseType" = { REPLACE = "" },
-"/SENTRY/X/prometheusExporter/description" = { REPLACE = "Template to consume metrics collected by the Prometheus Exporters.Needs customization depending on the Exporters and the Metrics collected." },
+"/SENTRY/X/prometheusExporter/description" = { REPLACE = "Template to consume metrics collected by a Prometheus Exporter.It needs customization depending on the Prometheus Exporter and the metrics collected." },
```
Comment (bertysentry) [inline templates/prometheus-exporter/prometheusExporterTemplate.cfg:6]: Missing space after first dot ;-)

File: templates/prometheus-exporter/README.md [hunk 2]
```diff
@@ -17,3 +17,5 @@ Add thresholds when appropriate.
 
 Import the .cfg file to add it to list of Templates.
 Set the %{ENDPOINT} and %{PORT} macros with the appropriate values.
+
+For more information about the Prometheus Exporter Template, refer to [Consuming Prometheus Metrics in Monitoring Studio X](https://www.sentrysoftware.com/support/user-documentation.html).
```
Comment (bertysentry) [inline templates/prometheus-exporter/README.md:21]: Please link to Monitoring Studio X's documentation: https://www.sentrysoftware.com/library/swsyx/index.html

**Review by bertysentry — CHANGES_REQUESTED** [review]
minor changes requested

File: templates/prometheus-exporter/README.md (hunk 2 above)
Comment (MarcDouguedroit) [reply templates/prometheus-exporter/README.md:21]: I added the link that @iguitton suggested.

**Review by bertysentry — APPROVED** [review]

File: templates/prometheus-exporter/README.md (hunk 2 above)
Comment (bertysentry) [reply templates/prometheus-exporter/README.md:21]: Please update the link (you can't label a link "Consuming Prometheus Metrics in Monitoring Studio X" and make it go to Sentry's list of all product docs).

**Review by bertysentry — CHANGES_REQUESTED** [review]
Please update the link

File: templates/prometheus-exporter/README.md (hunk 2 above)
Comment (iguitton) [reply templates/prometheus-exporter/README.md:21]: @bertysentry , the link https://www.sentrysoftware.com/library/swsyx/prometheus/consuming-prometheus-metrics-in-patrol.html will only be effective when the Monitoring Studio 10.3 documentation is published on Sentry Website. In the meanwhile, you will be redirected to the page listing all Sentry User Docs.

**Review by bertysentry — APPROVED** [review]
nice!

**Pull request merged**